source/_integrations/roborock.markdown

```diff
@@ -134,7 +134,7 @@ You can see all the maps within your Roborock account. Keep in mind that they ar
 
 ## Dyad entities
 
-Roborock wet/dry vacuums currently expose some entities through an MQTT connection - it is currently cloud dependent.
+Roborock wet/dry vacuums currently expose some entities through a MQTT connection - it is currently cloud dependent.
```

Comment:
It is actually `an`. This abbreviation starts with a vowel sound (`em cue tee tee`), so the correct article is `an` (regardless of the actual first letter in this case).

```suggestion
Roborock wet/dry vacuums currently expose some entities through an MQTT connection - it is currently cloud dependent.
```